### PR TITLE
AI Assistant: generate content from selected blocks

### PR DESCRIPTION
**Pull request by retrofox**
Fixes #

## Proposed changes:

* https://github.com/Automattic/jetpack/issues/30547

### Other information:

- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to the block editor
* Ensure you have a few paragraphs there
* Select multiple blocks
* Confirm you see the multiple blocks edition in the sidebar 
* Generate new content 

https://github.com/Automattic/jetpack/assets/77539/5a03bdc8-c52c-4491-a751-f95897a624bb

* When the content is unclear, the app will try to show a notice:

image

### CODE REVIEW

**Comment by oskosk** [comment]
Pushed rebase in 394027d36aa73022ee0b1ae93b75e5689733c5d2

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/generate-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+		key,
+		name: `${ PROMPT_TONES_MAP[ key ].emoji } ${ PROMPT_TONES_MAP[ key ].label }`,
+	};
+} );
+
+// toneOptions.unshift( {
+// 	key: '',
+// 	name: '',
+// } );
+
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/generate-panel.tsx]:
```suggestion
```

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/generate-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+		key,
+		name: LANGUAGE_MAP[ key ].label,
+	};
+} );
+
+// langOptions.unshift( {
+// 	key: '',
+// 	name: '',
+// } );
+
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/generate-panel.tsx]:
```suggestion
```

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/style.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.jetpack-ai-assistant__multiple-blocks-edition-panel {
+	.components-custom-select-control {
+		width: 100%;
+	}
+}
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/multiple-blocks-edition/style.scss]:
```suggestion
}

```

**Review by dhasilva — APPROVED** [review]
LGTM

**Pull request merged**